tensorflow_lite_support/ios/task/vision/utils/sources/GMLImage+Utils.h

```diff
@@ -53,13 +65,13 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Loads an image from a file in an app bundle into a GMLImage object.
  *
- * @param classObject The specified class associated with the bundle containing
- * the file to be loaded.
+ * @param classObject The specified class associated with the bundle containingthe file to be
```

Comment:
typo: containingthe

I think you're using a config with line width = 100 chars. Google's code require line width <= 80 chars so a lot of changes here are accidentally caused by that incorrect config.

Reply:
I have fixed the typo. Will the char length be resolved when you merge the PR?

Reply:
NVM I fixed the line width.